blog-service/2023-07-24-apps.md

```diff
@@ -0,0 +1,77 @@
+* [Cato Networks](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source/)
+* [Cisco Meraki](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source/)
+* [Citrix Cloud](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source/)
+* [Crowdstrike Falcon Data Replicator](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-fdr-source/)
+* [DocuSign Monitor](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/)
+* [Druva](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source/)
+* [Google BigQuery](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source/)
+* [KnowBe4](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source/)
+* [Microsoft O365 Exchange Trace Logs](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-trace-logs/)
+* [Rapid7 InsightVm](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source/)
+* [Webex](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source/)
+* [Zero Networks Segment Audit and Activities logs](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source/)
+
+And here is the list of new security apps added to the catalog:
+* [Airtable](/docs/integrations/saas-cloud/airtable/)
+* [Dropbox](/docs/integrations/saas-cloud/dropbox/)
+* [Microsoft Graph Azure Active Directory Reporting](/docs/integrations/microsoft-azure/active-directory-azure/)
+* [Microsoft Graph Identity Protection](/docs/integrations/microsoft-azure/microsoft-graph-identity-protection/)
+* [Microsoft Graph Security](/docs/integrations/saas-cloud/microsoft-graph-security-v2/)
+* [Microsoft O365 Exchange Trace Logs](/docs/integrations/saas-cloud/microsoft-exchange-trace-logs/)
+* [Mimecast](/docs/integrations/saas-cloud/mimecast/)
+* [Miro](/docs/integrations/saas-cloud/miro/)
+* [Proofpoint on Demand](/docs/integrations/saas-cloud/proofpoint-on-demand/)
+* [Proofpoint Targeted Attack Protection](/docs/integrations/saas-cloud/proofpoint-tap/)
+* [Palo Alto Cortex XDR](/docs/integrations/saas-cloud/palo-alto-cortex-xdr/)
+* [Rapid7](/docs/integrations/saas-cloud/rapid7/)
+* [SentinelOne](/docs/integrations/saas-cloud/sentinelone/)
+* [Symantec Web Security Service](/docs/integrations/saas-cloud/symantec-web-security-service/)
+
+Did you know Sumo Logic has an active community of users generating new content as well? In case you missed it, the [Sumo Logic Community Ecosystem](/docs/integrations/community-ecosystem-apps/) launched in February of this year and is the perfect conduit for security practitioners to share and rate open-source applications. Below is the current list of content from the Community Ecosystem:
```

Comment:
```suggestion
Did you know Sumo Logic has an active community of users generating new content as well? In case you missed it, the [Sumo Logic Community Ecosystem](/docs/integrations/community-ecosystem-apps/), which launched in February, is the perfect conduit for security practitioners to share and rate open-source applications. Below is the current list of content from the Community Ecosystem:
```